dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftLogsConnector.java

```diff
@@ -82,19 +82,26 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
         // DDL TEXT is simple ...
         // min() as there is no ANY() or SOME()
         String queryTemplateDDL
-                = " SELECT min(L.userid) as UserId, min(L.starttime) as StartTime, L.xid as xid, "
-                + " LISTAGG(CASE WHEN LEN(RTRIM(L.text)) = 0 THEN text ELSE RTRIM(L.text) END) WITHIN GROUP (ORDER BY L.sequence) as SqlText "
-                + " FROM STL_DDLTEXT L WHERE ## GROUP BY L.xid ";
+            = " SELECT any_value(L.userid) as UserId, any_value(L.starttime) as StartTime,"
+            + " any_value(L.endtime) as EndTime, any_value(trim(label)) as Label, L.xid as xid,"
+            + " L.pid as pid, LISTAGG(CASE WHEN LEN(RTRIM(L.text)) = 0 THEN text ELSE RTRIM(L.text)"
+            + " END) WITHIN GROUP (ORDER BY L.sequence) as SqlText FROM STL_DDLTEXT L WHERE ##"
+            + " GROUP BY L.xid, L.pid ";
```

Comment:
This formatting of the query is a bit difficult to read

Reply:
Agreed

Reply:
Done.